toolchain/sem_ir/node.h

```diff
@@ -400,7 +400,8 @@ class Node : public Printable<Node> {
       FactoryNoType<NodeKind::StructTypeField, StringId /*name_id*/,
                     TypeId /*type_id*/>;
 
-  using StructValue = Factory<NodeKind::StructValue, NodeBlockId /*refs_id*/>;
+  using StructLiteral =
+      Factory<NodeKind::StructLiteral, NodeBlockId /*refs_id*/>;
```

Comment:
Is it terrible for me to want these moved relative Type?

Reply:
Done.